src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -640,7 +640,7 @@ public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableAr
             Debug.Assert(method.ParameterCount == args.Length);
             return new BoundCall(
                 Syntax, receiver, method, args,
-                ImmutableArray<String>.Empty, ImmutableArray<RefKind>.Empty, false, false, false,
+                ImmutableArray<String>.Empty, default(ImmutableArray<RefKind>), false, false, false,
```

Comment:
Similarly, `argumentNamesOpt` value should be `default(ImmutableArray<string>)`, here and in method below.